app/frontend/wallet/blockchain-explorer.ts

```diff
@@ -376,9 +380,34 @@ const blockchainExplorer = (ADALITE_CONFIG) => {
     return response
   }
 
-  function getValidStakepools(): Promise<any> {
-    const url = `${ADALITE_CONFIG.ADALITE_BLOCKCHAIN_EXPLORER_URL}/api/v2/stakePools`
-    return request(url)
+  async function getValidStakepools(): Promise<Stakepools> {
```

Comment:
Please rebase, these have been already typed so you might need to fix this a bit.

Reply:
I rebased, not sure if I resolved everything correctly. I made these resolutions, for pool info:
- `Stakepool` - is used also outside of block chain explorer and has slightly different fields, so I kept it
- `StakePoolInfo` - is used in block chain explorer and not even exported outside of file, so I kept it
For mappings:
- `Stakepools` - since we changed the way mapping works, as in having an underlying object with two accessors `getPoolInfoByTicker` and `getPoolInfoByPoolHash`, I kept this type and removed the other type: `ValidStakePoolsMapping`, because this refactor cause to not be used anywhere anymore

Reply:
Makes sense.